package.json

```diff
@@ -61,7 +61,7 @@
     "@playkit-js/playkit-js-dash": "1.30.0",
     "@playkit-js/playkit-js-hls": "1.29.0",
     "@playkit-js/playkit-js-ui": "0.70.1",
-    "hls.js": "^1.0.7",
+    "hls.js": "^1.1.5",
```

Comment:
why do we actually have this both here and in the adapter ?

Reply:
it's a good question. I think the idea was to be able to upgrade the 3th party (hlsjs/shaka) without updating the adapter, but in fact we dont do it

Reply:
so can we remove it ?

Reply:
@OrenMe WDYT?